tsBindings/types.d.ts

```diff
@@ -105,7 +105,7 @@ export type SwapStatusErrorExpired = { token0_txn_out?: bigint | undefined, };
 export type PinNumberSettings = { length: number, attempts_blocked_until?: bigint | undefined, };
 export type ApproveError = { "BadFee": { expected_fee: bigint, } } | { "InsufficientFunds": { balance: bigint, } } | { "AllowanceChanged": { current_allowance: bigint, } } | { "Expired": { ledger_time: bigint, } } | "TooOld" | { "CreatedInFuture": { ledger_time: bigint, } } | { "Duplicate": { duplicate_of: bigint, } } | "TemporarilyUnavailable" | { "GenericError": { error_code: bigint, message: string, } };
 export type VideoCallPresence = "Default" | "Owner" | "Hidden";
-export type ChatMetrics = { text_messages: bigint, image_messages: bigint, video_messages: bigint, audio_messages: bigint, file_messages: bigint, polls: bigint, poll_votes: bigint, icp_messages: bigint, sns1_messages: bigint, ckbtc_messages: bigint, chat_messages: bigint, kinic_messages: bigint, deleted_messages: bigint, giphy_messages: bigint, prize_messages: bigint, prize_winner_messages: bigint, replies: bigint, edits: bigint, reactions: bigint, proposals: bigint, reported_messages: bigint, message_reminders: bigint, custom_type_messages: bigint, last_active: bigint, };
+export type ChatMetrics = { text_messages: bigint, image_messages: bigint, video_messages: bigint, audio_messages: bigint, file_messages: bigint, polls: bigint, poll_votes: bigint, crypto_messages: bigint, icp_messages: bigint, sns1_messages: bigint, ckbtc_messages: bigint, chat_messages: bigint, kinic_messages: bigint, deleted_messages: bigint, giphy_messages: bigint, prize_messages: bigint, prize_winner_messages: bigint, replies: bigint, edits: bigint, reactions: bigint, proposals: bigint, reported_messages: bigint, message_reminders: bigint, custom_type_messages: bigint, last_active: bigint, };
```

Comment:
This `crypto_messages` field can't go out on the frontend until groups and communities are upgraded

Reply:
Notice my comment - this PR is already waiting on groups and communities...

Reply:
Also can't go out until Users are upgraded, since they return metrics too